### PR TITLE
feat: [CDS-36450]: re-design Execution healthCard

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
##### Screenshots:

Earlier: 
image

Now:

https://user-images.githubusercontent.com/98325173/170483684-75305256-7e54-4a23-b835-fbe0f651fa2b.mov

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

File: src/modules/75-cd/pages/dashboard/CDDashboardPage.tsx [hunk 1]
```diff
@@ -111,7 +111,9 @@ export const CDDashboardPage: React.FC = () => {
     queryParams: {
       accountIdentifier: accountId,
       projectIdentifier,
-      orgIdentifier
+      orgIdentifier,
+      startTime: timeRange?.range[0]?.getTime() || 0,
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/pages/dashboard/CDDashboardPage.tsx:115]: Can we have a defaultTo here?

File: src/modules/75-ci/pages/dashboard/__test__/__snapshots__/CIDashboardPage.test.tsx.snap [hunk 2]
```diff
@@ -355,8 +355,9 @@ exports[`CIDashboardPage snapshot test should render properly 1`] = `
                 class="StyledProps--font StyledProps--main title"
               >
                 repositories
-                 
-                (1)
+                 (
+                1
+                )
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-ci/pages/dashboard/__test__/__snapshots__/CIDashboardPage.test.tsx.snap:360]: This is a CI dashboard snap. From the above changes, we are enabling the counts of the failed and active deployments. This is to be enabled on the CI front as well?

Comment (Sagar-Rajput7) [reply src/modules/75-ci/pages/dashboard/__test__/__snapshots__/CIDashboardPage.test.tsx.snap:360]: @AshwinBhatkal-Harness - no, it's just a formatting issue I guess
I hide these counts earlier - https://github.com/harness/harness-core-ui/pull/8749
their also this happened so now in this pr I just reverted back what I hide earlier

File: src/modules/75-cd/pages/dashboard/CDDashboardPage.tsx (hunk 1 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/pages/dashboard/CDDashboardPage.tsx:115]: yeah, we can, I did it in other places and forgot to do it here.

Comment (Sagar-Rajput7) [reply src/modules/75-cd/pages/dashboard/CDDashboardPage.tsx]: resolved

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Comment by Sagar-Rajput7** [comment]
retrigger coverage

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Review by samarth-sharma31 — APPROVED** [review]

**Review by vardanbansal-harness — DISMISSED** [review]

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Comment by Sagar-Rajput7** [comment]
retrigger jest

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Comment by Sagar-Rajput7** [comment]
retrigger eslint

**Review by vardanbansal-harness — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**